.github/workflows/gradle.yml

```diff
@@ -69,6 +76,20 @@ jobs:
     - name: Build with Gradle with local and optional remote DocumentDB tests
       run: |
         ./gradlew build -PrunRemoteIntegrationTests=$RUN_REMOTE_INTEGRATION_TESTS
+    - name: Extract gpg-key-pair into file
+      if: ${{ github.event.inputs.mavenFilesPreparation }}
+      run: |
+        echo "$GPG_KEY_SECRET_KEY_RING" > secring.kbx.b64
+        base64 -d secring.kbx.b64 > secring.kbx
+    - name: Build with Gradle with local and optional remote DocumentDB tests
+      if: ${{ github.event.inputs.mavenFilesPreparation }}
+      run: |
+        ./gradlew build publishToMavenLocal -Psigning.secretKeyRingFile=secring.kbx -Psigning.password=$GPG_KEY_PASSWORD -Psigning.keyId=$GPG_SIGN_KEY
+    - name: Copy Files for Maven
+      if: ${{ github.event.inputs.mavenFilesPreparation }}
+      run: | 
+        mkdir output/maven
+        cp -R ~/.m2/repository/software/amazon/documentdb/jdbc/documentdb-jdbc/* output/maven
```

Comment:
@affonsoBQ Did you resolve the issue of which outputs/JAR files to publish? Could it be fine-tuned here?

Reply:
fine-tuned to make the life easier in the manual publishing part